### PR TITLE
docs(cache): added instructions for flushing cache from disk

### PR DESCRIPTION
**Pull request by feroda**
I have documented to remove the ''system_cache'' and the ''views_simplecache'' directory
to flush the cache as a last resort, because other methods were
not applicable to me while migrating an ELGG installation to a new server:
- upgrade.php: did not flush
- I have no access to the ELGG admin panel

### CODE REVIEW

**Comment by jdalsem** [comment]
Lgtm

File: docs/guides/views/simplecache.rst [hunk 1]
```diff
@@ -24,6 +24,9 @@ You can regenerate the Simplecache at any time by:
 - In the admin panel click on 'Flush the caches'
 - Enabling or disabling a plugin
 - Reordering your plugins
+- Removing your ``datadir/system_cache``
```
Comment (mrclay) [inline docs/guides/views/simplecache.rst]: This is unrelated to simplecache.

**Comment by mrclay** [comment]
Better locations for these instructions:
https://github.com/Elgg/Elgg/blob/1.12/docs/admin/backup-restore.rst#bringing-it-all-together
https://github.com/Elgg/Elgg/blob/1.12/docs/admin/duplicate-installation.rst#run-upgrade-php

**Comment by feroda** [comment]
I have followed instructions here ;) https://github.com/Elgg/Elgg/pull/9645

I had problem only with system_cache if it is unrelated to simplecache then I put it somewhere else. @mrclay you pointed me to two locations, can you tell me which is the preferred one?

thank you guys for all your work

**Comment by mrclay** [comment]
Thanks for your work on this but I think #9763 would be a better approach. Docs help, but directly alerting users of the problem would be better.

**Comment by hypeJunction** [comment]
Closing in favor of #9764
